docs/en/latest/tutorials/add-multiple-api-versions.md

```diff
@@ -0,0 +1,227 @@
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## What is API versioning?
+
+**API versioning** is the practice of managing changes to an API and ensuring that these changes are made without disrupting clients. A good API versioning strategy clearly communicates the changes made and allows API consumers to decide when to upgrade to the latest version at their own pace.
+
+## Types of API Versioning
+
+#### URI Path
+
+The most common way to version an API is in the URI path and is often done with the prefix "v". This method employs URI routing to direct requests to a specific version of the API.
+
+```shell
+http://org.apisix/v1/hello
+http://org.apisix/v2/hello
+```
+
+#### Query Params
+
+In this method, the version number is included in the URI, but as a query parameter instead of in the path.
+
+```shell
+http://org.apisix/hello?version=1
+http://org.apisix/hello?version=2
+```
+
+#### Custom Request Header
+
+You can also set the version number using custom headers in requests and responses. This leaves the URI of your resources unchanged.
+
+```shell
+http://org.apisix/hello -H 'Version: 1'
+http://org.apisix/hello -H 'Version: 2'
+```
+
+The primary goal of versioning is to provide users of an API with the most functionality possible while causing minimal inconvenience. Keeping this goal in mind, let’s have a look in this tutorial at how to _publish and manage multiple versions of your API_ with Apache APISIX.
+
+**In this tutorial**, you learn how to:
+
+- Create a route and upstream for our sample API.
+- Add a new version to the existing API.
+- Use [proxy-rewrite](https://apisix.apache.org/docs/apisix/plugins/proxy-rewrite/) plugin to rewrite the path in a plugin configuration.
+- Route API requests from the old version to the new one.
+
+## Prerequisites
+
+For the demo case, we will leverage the sample repository [Evolve APIs](https://github.com/nfrankel/evolve-apis) on GitHub built on the Spring boot that demonstrates our API. You can see the complete source code there.
+
+To execute and customize the example project per your need shown in this tutorial, here are the minimum requirements you need to install in your system:
+
+- [Docker Desktop](https://docs.docker.com/desktop/windows/install/) - you need [Docker desktop](https://www.docker.com/products/docker-desktop/) installed locally to complete this tutorial. It is available for [Windows](https://desktop.docker.com/win/edge/Docker%20Desktop%20Installer.exe) or [macOS](https://desktop.docker.com/mac/edge/Docker.dmg).
```

Comment:
Maybe mention Docker instead of Docker Desktop? Linux does not have Docker Desktop. Ideally you want docker and docker compose installed. But now, I think compose installation comes with docker install.

Reply:
Changed it to Docker